dask/bytes/tests/test_hdfs.py

```diff
@@ -133,15 +185,17 @@ def test_read_text(hdfs):
     with hdfs.open('%s/text.2.txt' % basedir, 'wb') as f:
         f.write('Dan 400\nEdith 500\nFrank 600'.encode())
 
-    b = db.read_text('hdfs://%s/text.*.txt' % basedir)
-    result = b.str.strip().str.split().map(len).compute()
-    assert result == [2, 2, 2, 2, 2, 2]
-
     with hdfs.open('%s/other.txt' % basedir, 'wb') as f:
         f.write('a b\nc d'.encode())
 
+    b = db.read_text('hdfs://%s/text.*.txt' % basedir)
+    result = b.str.strip().str.split().map(len).compute(get=dask.get)
```

Comment:
There have been multiprocessing issues with hdfs3 in the past.  It might be wise to leave these with the multiprocessing scheduler.

Reply:
Fixed. Looks like there's fork-safety issues with libhdfs (or pyarrow's wrapper of it, not sure). Will file an issue. Testing using the spawn context works fine though, and mimics how things would work with the distributed scheduler.